modules/rosa-common-commands.adoc

```diff
@@ -19,6 +19,17 @@ Enables debug mode for the parent command.
 $ rosa create cluster --cluster=<cluster_name> --debug
 ----
 
+[id="rosa-download_{context}"]
+== download
+
+Downloads the latest compatible version of the specified software to the current directory in an archive file. Extract the contents of the archive and add the contents to your path to use the software. To download the latest Red Hat OpenShift Service on AWS CLI, specify `rosa`. To download the latest OpenShift Container Platform CLI, specify `oc`.
```

Comment:
I went with the expanded name here because in theory this sentence shouldn't vary - you only get the ROSA CLI if you specify 'rosa'. @bmcelvee @EricPonvelle Your thoughts appreciated here!

Reply:
@laubai The `{product-title}` variable will output "Red Hat OpenShift Service on AWS".

We usually use the introduce it with "`{product-title}` (ROSA)" on the first instance in the module, then just ROSA after.

Reply:
Understood, thankyou!